libs/dh/shared/feature-authorization/src/lib/dh-actor-token.service.ts

```diff
@@ -71,9 +73,19 @@ export class DhActorTokenService {
     return this.marketParticipantUserHttp.v1MarketParticipantUserGetUserActorsGet().pipe(
       tap(({ actorIds }) => this.actorStorage.setUserAssociatedActors(actorIds)),
       switchMap(() =>
-        this.tokenHttp
-          .v1TokenPost(this.actorStorage.getSelectedActor())
-          .pipe(map(({ token }) => token))
+        this.tokenHttp.v1TokenPost(this.actorStorage.getSelectedActor()).pipe(
+          map(({ token }) => token),
+          tap(async () => {
```

Comment:
question: does it have to be `async`?

Reply:
Nope, good catch, was a left over, from testing